cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsUtils.java

```diff
@@ -106,4 +110,20 @@ private static void setStringConfigIfExists(CruiseControlMetricsReporterConfig c
       // let it go.
     }
   }
+
+  /**
+   * Create a config altering operation if config's current value does not equal to target value.
+   * @param configsToAlter Set of config altering operations to be applied.
+   * @param configName Name of config to check.
+   * @param targetConfigValue Target value of the config.
+   * @param currentConfig Current value of the config.
+   */
+  public static void maybeUpdateConfig(Set<AlterConfigOp> configsToAlter,
+                                       String configName,
+                                       String targetConfigValue,
```

Comment:
Would it be possible to combine `configName` and `targetConfigValue` in a `Map`? This allows getting response from this util function in one call.